bin/add-deadlines-to-legacy-tasks

```diff
@@ -65,9 +74,13 @@ const migrate = async () => {
       .select('id')
       .whereIn('status', withAsruOrComplete)
       .whereJsonSupersetOf('data', { model: 'project', action: 'grant' })
-      .whereRaw("data->>'deadline' is null"); // will match both deadline: null and missing deadline key
+      .where(builder => {
+        // will match deadline: null, missing deadline key, and missing deadline->standard key
+        builder.whereRaw("data->>'deadline' is null")
+          .orWhereRaw("data->'deadline'->>'standard' is null");
+      });
```

Comment:
Because preprod now has the partial deadlines on some records, we need to fetch them in order to fix them.